examples/arc_01/run_pstd_bscan.m

```diff
@@ -56,3 +56,8 @@
 imagesc(dat_fs.z_i,dat_fs.x_i,abs(squeeze(dat_cyl.Ex_i)));
 title('Focussed beam with scattering cylinder');
 axis equal;
+
+%%
+figure;
+imagesc(dat_cyl.z_i, dat_cyl.x_i, abs(squeeze(dat_cyl.Hz_out)));
+axis equal;
```

Comment:
This makes a second image, does it?
+1 to leaving it in though, if it's the nice sphere one.

Reply:
It makes our sphere logo in fact!

Reply:
What is flavour text?
I mean sure: go nuts.